uwblueprint/jump-math
# set up user model and linting

## Notion ticket link

[Update user model](https://www.notion.so/uwblueprintexecs/Update-user-model-e1296e5a015c48479ceaf9c584acac57)
[Set up linting](https://www.notion.so/uwblueprintexecs/Setup-linting-81cd22db37254845a852a28a16c57dfe)

(sorry I combined 2 tickets into the same branch)


## Implementation description
* Linting: used [this commit](https://github.com/uwblueprint/rowan-house/commit/871dee67239406a87a4bbeafd46c506eb9d73f6b) as reference
* User Model: added email field and refactored the `Role` type



## Steps to test
1. Linting: check if code gets formatted when you push
2. User Model: check if the starter code still functions normally



## What should reviewers focus on?
* I renamed `Role: User | Admin` to `Role: TEACHER | ADMIN` and this had to be changed in a lot of places. Probably try to see if I missed any.


## Checklist
